### PR TITLE
Edit

### PR DESCRIPTION
**Pull request by hugh-burgess**

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -51,24 +51,25 @@ app.get("/courses/:courseId", (req, res) => {
...
   const { courseId } = req.params;
-  Student.find({ course: courseId }).then((students) => {
-    if (students.course !== courseId) {
+  Course.findById(courseId).then((course) => {
+    if (course) {
+      Student.find({ course: courseId }).then((students) => {
+        console.log(students);
+        res.json(students);
+        res.status(200);
+      });
```
Comment (marwinburesch) [inline server/app.js]: Looking at the endpoint `url` I think you only need this bit here. All students will be filtered based on the quey params, so you don't really need to look up any course

File: server/app.js [hunk 2]
```diff
@@ -51,24 +51,25 @@ app.get("/courses/:courseId", (req, res) => {
...
-      res.status(200);
     }
   });
 });
-
 // Get Single Student From Specific Course
 app.get("/courses/:courseId/students/:studentId", (req, res) => {
   const { studentId, courseId } = req.params;
-  Student.findById(courseId && studentId)
+  Student.find({ course: courseId, _id: studentId })
```
Comment (marwinburesch) [inline server/app.js:68]: Ha ha, nice solution to that weird task! Usability is questionable, but yeah that was the task :)

**Review by marwinburesch — CHANGES_REQUESTED** [review]
Excellent! 💯

File: server/app.js (hunk 2 above)
Comment (hugh-burgess) [reply server/app.js:68]: this was what Namir covered today and I implemented, thanks!

**Pull request merged**